conu/backend/docker/image.py

```diff
@@ -44,19 +46,37 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return super(DockerImageFS, self).__exit__(exc_type, exc_val, exc_tb)
 
 
+class DockerImagePullPolicy(enum.Enum):
+    NEVER = 0
+    IF_NOT_PRESENT = 1
+    ALWAYS = 2
```

Comment:
One can find in ALWAYS policy promise that the actual pulling would happen every time (not just contacting docker API). We might want to specify in documentation why pulling image, that is already present, wouldn't be beneficial so there is no confusion.